frameworks/helloworld/tests/test_region_awareness.py

```diff
@@ -14,18 +14,30 @@
 POD_NAMES = ['hello-0', 'world-0', 'world-1']
 
 
+def with_service_installation(additional_options):
```

Comment:
This could just be a `pytest.fixture`. The default scope is per test. https://stackoverflow.com/questions/22627659/run-code-before-and-after-each-test-in-py-test

(These can also be parameterized).